exploration/u-options.md

```diff
@@ -0,0 +1,149 @@
+# `u:` Options
+
+Status: **Accepted**
+
+<details>
+	<summary>Metadata</summary>
+	<dl>
+		<dt>Contributors</dt>
+		<dd>@eemeli</dd>
+		<dt>First proposed</dt>
+		<dd>2023-05-06</dd>
+		<dt>Pull Requests</dt>
+		<dd><a href="https://github.com/unicode-org/message-format-wg/pull/780">#780</a></dd>
+	</dl>
+</details>
+
+## Objective
+
+Provide common definitions for contextual options.
+
+## Background
+
+Function options may influence the resolution, selection, and formatting of annotated expressions.
+These provide a great solution for options like `minFractionDigits`, `dateStyle`,
+or other similar factors that influence the formatted result.
+
+However, this single bag of options is not appropriate in all cases,
+in particular for options that change the formatting context for the function.
+
+## Use-Cases
+
+At least the following function and markup options should be considered:
+
+- An identifier for the expression or markup.
+  This is included in the formatted part,
+  and allows the parts of an expression to be explicitly addressed.
+
+  > Example identifying two literal numbers:
+  >
+  > ```
+  > The first number was {1234 :number u:id=first} and the second {56789 :number u:id=second}.
+  > ```
+
+- An override for the locale used to format the expression.
+  Should be expressed as a non-empty sequence of BCP 47 language codes.
```

Comment:
Sequence?

```suggestion
The value of this option is a Unicode Locale Identifier
(which is a BCP47 language tag).
The root locale is expressed using the tag `und`.
```

Reply:
I copied this from the expression-attributes text. As this is describing a use case, it's perhaps best to not get stuck in the technical details here.
```suggestion
```